server/guides/other/domains.md

```diff
@@ -0,0 +1,23 @@
+# Domains
+
+This article aims to provide you with the knowledge to connect your own domains to your own Minecraft servers.
+
+## Obtaining a Domain
+
+Domains can be purchased from a wide range of venders some of these include; [Godaddy](https://godaddy.com), [Namecheap](https://www.namecheap.com) and [Google](https://domains.google/). You can also purchase your domain from any vender you prefer.
+
+## DDOS Protection (Optional)
+
+DDOS (Distributed Denial of Service) attacks flood the router/modem with so many requests that it cant respond to all of them causing your Minecraft server and sometimes the rest of your internet to go down. Some domain vendors offer a DDOS protection service for free or an added item at checkout.
+
+### Cloudflare
+
+To prevent this proxies route the traffic through their servers taking the load off your network. To protect your server from DDOS attacks you can use a service like [cloudflare](https://www.cloudflare.com/) which offers free and paid tiers, to protect a Minecraft server the free tier is adequate. After you have created a cloudflare account you can add your domain to the free tier using the "Add a Site" button.

```

Comment:
**Cloudflare** is a name and should be written with a capital **C**.

Minecraft-DDoS-Protection is not included in the free Cloudflare tier. Cloudflare only protects websites for free. Look out for **Spectrum** [here](https://www.cloudflare.com/plans/#overview).

Consider adding **TCPShield** as a free example for Minecraft-Server DDoS mitigation.